be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -514,6 +519,15 @@ object DbActor {
    */
   final case class DbActorReadLaoDataAck(laoData: LaoData) extends DbActorMessage
 
+  /**
+   * Response for a [[AddWitnessSignature]] db request Receiving [[DbActorAddWitnessMessage]] works as
+   * an acknowledgement that the request was successful
+   *
+   * @param witnessMessage requested message witnessed
+   */
+  final case class DbActorAddWitnessMessage(witnessMessage: Message) extends DbActorMessage
```

Comment:
Should be renamed to DbActorAddWitnessSignatureAck  (or the name should contain Ack as others Acks)

Reply:
To me, it does not seem strictly necessary but could be good for protocol consistency, it could be discussed quickly between you